docs/cell_census_schema_0.2.0.md

```diff
@@ -667,12 +667,14 @@ For each organism the `SOMAExperiment` MUST contain the following:
 	* Matrix  data –  `census_obj[“census_data”][organism].ms[“RNA”].X` – `SOMACollection`. It MUST contain exactly one layer: 
 		* Count matrix – `census_obj[“census_data”][organism].ms[“RNA”].X[“raw”]` – `SOMASparseNDArray`
 	* Feature metadata – `census_obj[“census_data”][organism].ms[“RNA”].var` – `SOMAIndexedDataFrame`
-	* Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”].varp[“dataset_presence_matrix”]` – `SOMASparseNDArray`
+	* Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”][“feature_dataset_presence_matrix”]` – `SOMASparseNDArray`
 
 
 For each organism the `SOMAExperiment` SHOULD NOT contain the following:
 
 * Elements of `census_obj[“census_data”][organism].ms[“RNA”]`: 
+	* `varm`
+	* `varp`
```

Comment:
do we want to note as being empty?

Reply:
Further - these are optional, and IMHO we should say nothing about them unless we actually use them....

Or put another way, what exactly does the above imply? That the schema _requires_ that we have an empty collection with those names?  That seems not useful...

Reply:
I agree, I have deleted that section now